scipy/stats/_precompute/skewnorm_generate_noncentral_moments.py

```diff
@@ -0,0 +1,41 @@
+# This script generates the Python code for the dictionary
+# of polynomial coefficients that is used to implement the method
+# _munp(order, a) for the skewnorm distribution in scipy.stats.
+#
+# This script is not part of the build system.  This script was
```

Comment:
Thanks for the explanation, I was confused at first...

Reply:
I followed @person142's example from `special` and tucked this away in the new `_precompute` directory.  The idea is to stash here the one-off scripts (typically based on sympy or mpmath) that we use to generate other bits of code.

Reply:
The repo is the *best* place to keep this.   It is not noise, it is a code generation tool.   Future devs (including our future selves) will thank us for keeping this code around. More than once, I've wished I had the code that was used to generate the various polynomials and rational functions that are used throughout `scipy.special`.  Having it here will allow future devs to verify the coefficients, and to possibly use variations of the script to generate code for other distributions.

I had used the Unicode character δ in the script because it made the output that is printed to the terminal look nice, but it is not important, so I switched it to `d`.  (There is actually no reason we couldn't allow δ in the Unicode check; we already have a few of the other Greek letters that are commonly used in mathematics.  But that can wait until someone finds a compelling reason to use it in a public docstring.)

Reply:
OK, I removed the script.   I'm fine with storing it somewhere else.  We can create new top-level repo for code generation and analysis scripts, useful docs, and anything else that we use for SciPy development that is worth saving but that doesn't quite fit into the structure of the main repo.  Who has the appropriate permissions for creating such a repo?

Reply:
Thank you Warren. All maintainers should be able to create a repo. Seems like I can, so I am sure you can too. We currently have archive. Maybe that could go there? I am fine with having a new one as well. Maybe this calls for an email on the list.